client/header/activity-panel/unread-indicators.js

```diff
@@ -31,21 +40,15 @@ export function getUnreadNotes( select ) {
 
 	// @todo This method would be more performant if we ask only for 1 item per page with status "unactioned".
```

Comment:
These comments are irrelevant now or they should be moved next to the `const`. I prefer just removing todos that will likely never be addressed but up to you how you tackle this.